assets/erc-7730/erc7730-v1.schema.json

```diff
@@ -362,7 +349,7 @@
 
                 "formats": {
                     "title": "List of field formats",
-                    "description": "The list includes formatting info for each field of a structure. This list is indexed by a key identifying uniquely the message's type in the abi. For smartcontracts, it is the selector of the function or its signature; and for EIP712 messages it is the primaryType of the message.",
+                    "description": "The list includes formatting info for each field of a structure. This list is indexed by the full function signature with parameter names.",
```

Comment:
Missing the index type for EIP-712 (since they do not have an equivalent of human readable ABI)